lib/process-services/task-list/components/task-header.component.ts

```diff
@@ -86,7 +86,8 @@ export class TaskHeaderComponent implements OnChanges, OnInit {
                     {
                         label: 'ADF_TASK_LIST.PROPERTIES.PRIORITY',
                         value: this.taskDetails.priority,
-                        key: 'priority'
```

Comment:
The code is fine. I found this typo isEditble can you fix it, please? (card-view-textitem.component.ts) there are 4 occurrences

Reply:
Can you add a unit test, please?

Reply:
Fixed the typo;
Added unit test